test/integration/server-discovery-and-monitoring/server_discovery_and_monitoring.spec.test.ts

```diff
@@ -3,11 +3,17 @@ import * as path from 'path';
 import { loadSpecTests } from '../../spec';
 import { runUnifiedSuite } from '../../tools/unified-spec-runner/runner';
 
+const skipTable: { pattern: string; reason: string }[] = [
+  { pattern: 'Topology lifecycle', reason: 'see NODE-5723' },
+  { pattern: 'connect with serverMonitoringMode=stream >=4.4', reason: 'NODE-6045' },
+  { pattern: 'connect with serverMonitoringMode=auto >=4.4', reason: 'NODE-6045' }
```

Comment:
We usually use the pattern `TODO(NODE-xxxx)` in our skip reasons - can we update them?

Also, can we describe why the test is skipped instead of just linking a jira ticket?  That's easier to grok when scanning test output.